lib/cppapi/InterceptPlugin.cc

```diff
@@ -88,6 +88,7 @@ struct InterceptPlugin::State {
   {
     plugin_mutex_ = plugin->getMutex();
     http_parser_  = TSHttpParserCreate();
+    plugin_       = plugin;
```

Comment:
We should probably set `plugin_` in the member initializer list, like `cont_`. Maybe others too but I'm not so sure.

Reply:
@maskit done